pkg/helpers/helpers_test.go

```diff
@@ -221,3 +222,75 @@ func Test_DecodeSnowflakeAccountIdentifier(t *testing.T) {
 		require.ErrorContains(t, err, fmt.Sprintf("unable to read identifier: %s", id))
 	})
 }
+
+func TestParseRootLocation(t *testing.T) {
+	tests := []struct {
+		name     string
+		location string
+		id       string
```

Comment:
id -> expectedId
path -> expectedPath
(we should have a clear separation between the setup attrs and the expected output)

Reply:
Done.